gradle.properties

```diff
@@ -1,4 +1,4 @@
-ideaVersion = IC-2018.1
+ideaVersion = IC-2018.2
```

Comment:
Not a big fan of upversioning but if this is absolutely required for Java11 then I'm fine as we can override

Reply:
yes it is required, before IC-2018.2 gradle build will fail with java 11